### ISSUE TITLE
Document required file formats for support by S-100 ECDIS.

### ISSUE THREAD

**Issue by kusala9** [iho-ohi/98-interoperability#7]
duplicate of https://github.com/iho-ohi/S-164-Sub-Group/issues/19 Once the S-164 issue is complete, this one can be used to track update of the documentation in S-98 Annex C.

**Comment by rmalyankar** [comment]
CSS files are also needed to mitigate the maintenance issue of having the same CSS code in multiple files. For 5.0/5.1 they would have to be treated as "other" support files in discovery metadata.

New draft clause added to Annex C.
